e2e/rdctl.e2e.spec.ts

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright © 2022 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/**
+ * This file includes end-to-end testing for the HTTP control interface
+ */
+
+import fs from 'fs';
+import path from 'path';
+
+import { expect, test } from '@playwright/test';
+import { BrowserContext, ElectronApplication, Page, _electron } from 'playwright';
+
+import fetch from 'node-fetch';
+import { createDefaultSettings, kubectl, playwrightReportAssets } from './utils/TestUtils';
+import { NavPage } from './pages/nav-page';
+import paths from '@/utils/paths';
+import { ServerState } from '@/main/httpCommandServer';
```

Comment:
I'm importing the _type_ here; we still read `rd-engine.json` for the actual values at runtime.

And no, I still can't test shutdown (trying to do so just hangs the test, because it's expecting responses from the app process).

Reply:
Ah, right. Everything else looked good.